datastacks/config_class.py

```diff
@@ -0,0 +1,55 @@
+from datetime import date
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class DataSourceTypes(Enum):
```

Comment:
Sorry for being so picky... Can we have it singular?

Reply:
Done